.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Starting with the most basic config. We can edit as and when required.

Reply:
We already know that these will be necessary:

      # K8s and operator SDK, we need to handle these manually
      - dependency-name: github.com/operator-framework/*
      - dependency-name: k8s.io/*
      - dependency-name: sigs.k8s.io/*